python/protobuf.c

```diff
@@ -68,34 +68,79 @@ PyUpb_ModuleState *PyUpb_ModuleState_Get(void) {
   return PyUpb_ModuleState_GetFromModule(module);
 }
 
+PyObject *PyUpb_GetWktBases(PyUpb_ModuleState *state) {
+  if (!state->wkt_bases) {
+    PyObject *wkt_module =
+        PyImport_ImportModule("google.protobuf.internal.well_known_types");
+
+    if (wkt_module == NULL) {
+      return false;
+    }
+
+    state->wkt_bases = PyObject_GetAttrString(wkt_module, "WKTBASES");
+    PyObject *m = PyState_FindModule(&module_def);
+    // Make sure it is GC'd.
```

Comment:
is or isn't?

Reply:
"is" is correct. `PyObject_GetAttrString()` will yield a reference that we own. This `PyModule_AddObject()` will take ownership of that reference, keeping it alive until the module itself is destroyed.

Reply:
can you rephrase the comment to "// Reparent ownership to `m`"